### PR TITLE
Create LOC7369Malqa.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
Created new ID for Malqa in Egypt. I suspect Malaqah in Dumyat as a possible identification, but I am not sure about that.

### CODE REVIEW

**Comment by CarstenHoffmannMarburg** [comment]
After having read the Guidelines on places, I still do not know what can I add or change.

**Comment by eu-genia** [comment]
Please read through my corrections to Tiberias and do exactly the same here. Thank you.

**Comment by CarstenHoffmannMarburg** [comment]
I don't see any opportunity for change:
You wrote: "for non-Ethiopian items (persons, places, works, etc.) the main title should always be given in English"
-> the identification of Malqa with Malaqa near Damyette/Dumyat is douptful and not safe.
You wrote: "In case the record remains this must be corrected extended according to the Guidelines https://betamasaheft.eu/Guidelines/?q=wikidata&id=places, see example 3, with the transcription, translation, (English marked as main), sameAs should be pointing to wikidata entry (and BM ID added to wikidata after the entry has been corrected and merged), location pointing at least to Israel/Palestine and better also to the region LOC3221Galile"
-> this does not apply to the case of Malqa, because we can't attribute it with certainty to any known city or topographical element.
Did you make any changes or request beside these, that I did not realize?

**Comment by eu-genia** [comment]
Please read very carefully the emails that you receive and/or the comments on GitHub
https://github.com/BetaMasaheft/Places/pull/84/files

I listed the necessary corrections, the changes are up to you. Thank you very much!

**Comment by eu-genia** [comment]
the only thing that does not apply here is wikidata if we cannot identify the place, all other recommendations apply

File: new/LOC7369Malqa.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LOC7369Malqa" xml:lang="en" type="place">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>ማልቃ</title>
```
Comment (eu-genia) [inline new/LOC7369Malqa.xml:8]: Title not in fidel

File: new/LOC7369Malqa.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-02-01">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="city">
+                    <placeName>ማልቃ</placeName>
```
Comment (eu-genia) [inline new/LOC7369Malqa.xml:48]: placeName n1 in fidel, corresp transcribed, translation if applicable, note on possible identification, country Egypt

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LOC7369Malqa.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply new/LOC7369Malqa.xml:48]: attribute 'n1' not allowed here: <placeName n1="">ማልቃ</placeName>
attribute 'corresp' demands for specific ID: <placeName corresp="">ማልቃ</placeName> (value of attribute "corresp" is invalid).
I don't know how to insert a translation.
Inserted a note: <note>Possibly the city of Malaqa near Dumyāṭ in the Nile delta.</note>, despite I think the identification with this city is more hypothetical. I do not even know whether this location even existed already in the 18th century.

File: new/LOC7369Malqa.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/LOC7369Malqa.xml:8]: Changed title to "Mālqā".

**Comment by eu-genia** [comment]
just a reminder, this record has to be corrected

**Review by eu-genia — APPROVED** [review]

**Pull request merged**